chainerx_cc/chainerx/python/core_module.cc

```diff
@@ -57,17 +62,23 @@ void InitChainerxModule(pybind11::module& m) {
     testing::testing_internal::InitChainerxTestingModule(m_testing);
 
     // Modifies __doc__ property of a pybind-generated function object.
-    m.def("_set_pybind_doc", [](py::handle obj, std::string docstring) {
+    m.def("_set_pybind_doc", [](py::handle obj, const std::string& docstring) {
         if (!py::isinstance<py::function>(obj)) {
             throw py::type_error{"Object is not a function."};
         }
 
         // This function is called only sequentially from Python module.
         // No need of race guard.
-        static std::vector<std::string>* docstrings_keeper = new std::vector<std::string>{};
+        static auto docstrings_keeper = new std::vector<const char*>{};
```

Comment:
Do you still need the `docstrings_keeper` ?

Reply:
Removed.